tests/unittests/cli/test_cli.py

```diff
@@ -13,7 +13,6 @@
 KEYSTORE_PATH = Path(__file__).resolve().parent.joinpath("keystore")
 SCENARIO = f"{Path(__file__).parent.joinpath('scenario', 'join-network-scenario-J1.yaml')}"
 CLI_ARGS = (
-    f"--chain goerli:http://geth.goerli.ethnodes.brainbot.com:8545 "
```

Comment:
please add a unit test that ensures the cli flag is still functional and respected

Reply:
I see, that you added https://github.com/raiden-network/scenario-player/pull/401/files#diff-682d38b401009dead7fde5128615d0b5R89 ( `test_chain_property_prioritizes_cli_attr_over_scenario_definition` ) -- but I'm afraid this is still prone to later regressions.